analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -96,7 +96,8 @@ class Analyzer(private val config: AnalyzerConfiguration, private val labels: Ma
             PackageManager.findManagedFiles(
                 absoluteProjectPath,
                 distinctPackageManagers,
-                config.excludes(repositoryConfiguration)
+                config.excludes(repositoryConfiguration),
+                repositoryConfiguration.includes
```

Comment:
For consistency with excludes, it would be better if includes would also be ignored here if `skipExcluded` is `false`.